lib/scraped_page_archive.rb

```diff
@@ -14,41 +14,100 @@
 class ScrapedPageArchive
   class Error < StandardError; end
 
-  attr_writer :github_repo_url
+  class GitStorage
+    attr_writer :github_repo_url
```

Comment:
The README documents that we allow people to set this using:

```
ScrapedPageArchive.github_repo_url = REPO
```

Is that still going to work if this is moved to here?